Mage.Sets/src/mage/cards/b/BrainstealerDragon.java

```diff
@@ -99,7 +99,7 @@ public boolean apply(Game game, Ability source) {
         for (Card card : cards.getCards(game)) {
             CardUtil.makeCardPlayableOrCastable(
```

Comment:
It's a good practice to make two methods for different rules/abilities -- one for "you may play cards" and another for "you may cast spells". With inner usage of shared method like `makeCardPlayableOrCastable`.